app/views/auth.html

```diff
@@ -11,11 +11,17 @@
     </div>
 
     <div class="loginFields">
+<label>
       <input type="text" ng-model="user.email" class="form-control" placeholder="Email Address">
-
-      <input type="password" ng-model="user.password" class="form-control" placeholder="Password">
-      <input type="password" ng-model="user.confirmpassword" ng-show="amRegistering"  class="form-control" ng-class="{fieldInvalid: (user.password != user.confirmpassword)}" placeholder="Confirm Password">
-  </div>
+</label>
+<label>  
```

Comment:
Make sure you be mindful of the spacing/tabs for the tags.

Following the indentation of the code makes it easier to read.